src/js/page/ui/view-toggler.js

```diff
@@ -16,13 +16,11 @@ export default class ViewToggler {
       // stop browsers remembering previous form state
       this.container.output[0].checked = true;
 
-      this.container.addEventListener('change', () => this._onChange());
```

Comment:
I can revert this if you prefer and I agree this might be consistent with the rest of the codebase. I just noticed that the minifier doesn't inline the functions used once. I didn't check any other places, just happened to notice this one because the file is small.